<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiPosition</class>
 <widget class="QWidget" name="GuiPosition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Position</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tab">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="position">
      <attribute name="title">
       <string>Positions</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutPosition">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="TablePosition" name="tablePosition">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>volume</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>SL</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>TP</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>current</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>diff</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>profit</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>pair</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>exchange</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>open date</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="balance">
      <attribute name="title">
       <string>Balance</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutBalance">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableBalance">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>currency</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>balance</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="orders">
      <attribute name="title">
       <string>Orders</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutOrders">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="TableOrder" name="tableOrders">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>volume</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>pair</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TablePosition</class>
   <extends>QTableWidget</extends>
   <header>TablePosition.h</header>
  </customwidget>
  <customwidget>
   <class>TableOrder</class>
   <extends>QTableWidget</extends>
   <header>TableOrder.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
